libraries/abstractions/ble_hal/test/include/iot_test_ble_hal_common.h

```diff
@@ -58,12 +58,20 @@
 
 /* Enable/Disable test cases */
 /* AFQP test */
-#define ENABLE_TC_WRITE_LONG              ( 1 )
-#define ENABLE_TC_ADD_INCLUDED_SERVICE    ( 0 )
-#define ENABLE_TC_SECONDARY_SERVICE       ( 0 )
+#define ENABLE_TC_WRITE_LONG                                      ( 1 )
```

Comment:
We can possibly move these configs to a separate header file per board, so that we can enable tests per board.
Sample per board configs for ESP can be found here: https://github.com/aws/amazon-freertos/tree/master/vendors/espressif/boards/esp32/aws_tests/config_files
A default configuration header file can be provided in  libraries/abstractions/ble_hal/test/include folder.